src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -342,13 +362,15 @@ type FSharpLspServer(backgroundServiceEnabled: bool, state: State, lspClient: FS
                             | FSharp.Analyzers.SDK.Info -> DiagnosticSeverity.Information
                             | FSharp.Analyzers.SDK.Warning -> DiagnosticSeverity.Warning
                             | FSharp.Analyzers.SDK.Error -> DiagnosticSeverity.Error
-                        { Diagnostic.Range = range
-                          Code = None
+                        { Range = range
+                          Code = Option.ofObj m.Code
                           Severity = Some s
-                          Source = sprintf "F# Analyzers (%s)" m.Type
+                          Source = $"F# Analyzers (%s{m.Type})"
                           Message = m.Message
                           RelatedInformation = None
-                          Tags = None }
+                          Tags = None
+                          CodeDescription = None
+                          Data = match m.Fixes with [] -> None | some -> Some (box some) }
```

Comment:
this specifically isn't useful right now, but I wanted to add it to start looking at the mechanism for round-tripping fixes.

Reply:
nevermind, just going ahead and doing this now, since it cleans up a lot of bookkeeping.